src/NServiceBus.NHibernate/ScriptGenerator.cs

```diff
@@ -112,6 +113,7 @@ static string GenerateScript(Configuration config)
             var formatter = FormatStyle.Ddl.Formatter;
             var script = new StringBuilder();
             export.Create(s => script.Append(formatter.Format(fixUpHelper.FixUp(s))), false);
+            script.Replace(global::System.Environment.NewLine, "\r\n");
```

Comment:
Why has this been done? We shouldn't be forcing people to use Windows line endings if they aren't on Windows.

Reply:
This has been done to make tests independent on the environment and it has been applied in wrong place. Will move it to the test suite